### DISCUSSION TITLE
Every weekend should be double XP

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3594] [Suggestions]
Why do we have to bother with working out which weekends are double xp. People have better things to do than reserve specific weekends for levelling.

Just make every weekend double XP. anyone who doesnt like it can just opt out .

Edit: looks like this is being implemented https://www.chromiecraft.com/en/an-august-update-aq20-war-effort-r12-14-gear-and-more/

**Comment by Saikur0n** [comment]
I would suggest reverse. Let it leave x1 blizzlike experience and give an option on weekend to enable double XP instead of making it by default on weekends double XP.

**Reply by BeefSteaker** [reply]
I think that'll just cause new players who would want it enabled to miss out. 

The small amount of people that do want it off are more likely to know they can turn it off than those who would want it on

**Reply by Saikur0n** [reply]
But in the project description it's clearly stated that it is Blizzlike rates and progression. They also can add information in the chat and web page how to make double exp in weekends. I like x1 rates philosophy because you play in a pace which game was designed by doing every quest and at the same time catching bugs on this server and reporting them to be fixed. There are dalaran wow, warmane, stormforge and wow circle which has higher rates.

**Reply by balleny** [reply]
or sticking with the server description and keeping the rates at 1x constantly, except for special occasions (winter holidays, etc)

**Reply by Saikur0n** [reply]
> or sticking with the server description and keeping the rates at 1x constantly, except for special occasions (winter holidays, etc)

Completely agree. There are a lot of servers with increased rates for those who bored from leveling up and feeling vibes of Azeroth and its lands during leveling. I really like that leveling is x1.

**Comment by BeefSteaker** [comment]
Looks like this is being Implemented https://www.chromiecraft.com/en/an-august-update-aq20-war-effort-r12-14-gear-and-more/

**Comment by Gultask** [comment]
@Saikur0n the command to disable the increased xp rate is `.weekendxp rate 1`
